ui/yarn.lock

```diff
@@ -12735,6 +12725,21 @@ __metadata:
   languageName: node
   linkType: hard
 
+"ember-cli-clipboard@npm:^1.0.0":
```

Comment:
Version 0.16.0 of ember-cli-clipboard (just above this new one) will be removed when the adoption of the <CopyButton> is complete. For now, this PR just adds the upgraded ember-cli-clipboard version 1.0.0